src/globus_cli/commands/group/invite/_common.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import uuid
+from typing import Dict, List, Optional
+
+import click
+import globus_sdk
+
+from globus_cli.parsing import ParsedIdentity
+from globus_cli.services.auth import CustomAuthClient
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+
+def get_invite_formatter(case: Literal["accept", "decline"]):
```

Comment:
```suggestion
def get_invite_formatter(case: Literal["accept", "decline"]) -> Callable:
```

Reply:
Yeah, perhaps I shouldn't be so lazy about annotations if I'm going to include them. I'll have to add the imports as well, and I'll make it `Callable[[GlobusHTTPRepsonse], None]`.